File: IO.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{

    freopen("IO_in1.txt", "r", stdin);
    int Testcases;
    scanf("%d", &Testcases); // Number of test cases
    while (Testcases--)
    { // Repeat until 0
        int a, b;
        scanf("%d %d", &a, &b);
        printf("%d\n", a + b);
    }

    return 0;
}

File: IO.java
import java.util.*;

class IO {
    public static void main(String[] args) {

        Scanner answer = new Scanner(new File("IO_in1.txt"));

        int TestCases = answer.nextInt();
        while (TestCases-- > 0) {
            int a = answer.nextInt(), b = answer.nextInt();
            System.out.println(a + b);
        }
    }
}